schema/meter_provider.json

```diff
@@ -75,6 +87,27 @@
                 }
             }
         },
+        "MetricProducer": {
+            "type": "object",
+            "additionalProperties": true,
+            "minProperties": 1,
+            "maxProperties": 1,
+            "properties": {
+                "opencensus": {
+                    "type": "object",
+                    "additionalProperties": false
+                },
+                "prometheus": {
```

Comment:
I wasn't sure whether to include the prometheus bridge here.  AFAIK this only exists in Go (maybe java too?).  Are properties added here required for implementations?  Or are they optional, and just a way to standardize the keys?

Reply:
We only add schema surface area for components that are explicitly defined in the spec. The go implementation of this producer should be registered as a [ComponentProvider](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/file-configuration.md#component-provider).

Reply:
I removed this component